web/client/components/widgets/builder/wizard/ChartWizard.jsx

```diff
@@ -143,6 +143,8 @@ const ChartWizard = ({
     if (!selectedTrace) {
         return null;
     }
+    const selectedTab = selectedTrace?.type === "pie" && tab === "axis" ? "traces" : (tab);
```

Comment:
I think we need to pass [tab={selectedTab}](https://github.com/geosolutions-it/MapStore2/blob/b18c868bc6dc83821db512af83843f1c83e3f7c1/web/client/components/widgets/builder/wizard/ChartWizard.jsx#L160) also to ChartTraceEditSelector

Reply:
done